spring-cloud-huawei-router/src/main/java/com/huaweicloud/router/client/loadbalancer/ZoneAwareServiceInstanceFilter.java

```diff
@@ -34,20 +33,20 @@
 
 public class ZoneAwareServiceInstanceFilter implements ServiceInstanceFilter {
 
-  private ServiceCombRegistration serviceCombRegistration;
+  private MicroserviceInstanceService instance;
```

Comment:
Can not replace `Registation` to `MicroserviceInstanceService`.

Reply:
Modified